docs/process/release-planning.md

```diff
@@ -1,7 +1,71 @@
 # Release Planning
 
-We plan work at two levels of granularity: first at the marketing release level (1.1, 1.2, etc.) and then at the milestone level. Marketing releases are made up of multiple milestones. Milestones should contain roughly a month’s worth of work.
+This document outlines our process for planning and scheduling releases, so you
+can familiarize yourself with the flow of work from opening an issue to seeing
+it published in a release.
 
-We’ll release updates as needed or when a milestone is completed.
+## Releases
 
-Large features and user-visible changes are feature-flagged, to be unflagged in the next marketing release. Beta and dev builds will have all feature flags enabled.
+We organize releases in two ways - marketing and milestones:
+
+ - Marketing releases are what we use to represent planned features, and are
+   high-level goals
+      - **for example: 1.4, 1.5, etc.**
+ - Milestone releases are used to track issues and pull requests associated
+   with a marketing release, and can be followed along on GitHub
+      - **for example: 1.4.1, 1.4.2, 1.5.0, etc.**
```

Comment:
i’m a little confused with 1.5.0 as an example. are we saying that “1.5” are “1.5.0" are intentionally used differently colloquially to differentiate between whether something is high level vs. something technically belonging in a milestone?

Reply:
Perhaps `1.5` is the title of the blog post and it represents the entire `1.5.x` line, whereas `1.5.0` is only the very first production release in that line?

Reply:
I made a change in cce8e9d874b059f537edb4856ab7739b76596c8f to avoid the potential confusion of "milestone releases" and instead just use GitHub milestones for this bit:

> Milestones are used to track issues and pull requests associated with a marketing release, and can be [followed on GitHub](https://github.com/desktop/desktop/milestones)

Reply:
And I just went back-and-forth with @billygriffin on this and I think this tweak was the bit that was missing:

> Milestones are used to track issues and pull requests associated with **an upcoming release**, and can be followed on GitHub